com.unity.netcode.gameobjects/Runtime/Messaging/Messages/ClientConnectedMessage.cs

```diff
@@ -73,6 +73,8 @@ public void Handle(ref NetworkContext context)
                     /// DANGO-TODO: Determine if this needs to be removed once the service handles object distribution
                     networkManager.RedistributeToClients = true;
                     networkManager.ClientsToRedistribute.Add(ClientId);
+
+                    // TODO: We need a client synchronized message or something like that here
```

Comment:
Is this worth tracking somewhere?

Reply:
Yeah, I need to add that to our list.
_(thank you for reminding me)_